roller.go

```diff
@@ -576,6 +658,10 @@ func main() {
 
 	wg.Wait()
 
+	if state.clusterAutoscaler.enabled == true {
+		enableClusterAutoscaler(state)
```

Comment:
the comparison to true can be left off with the same affect

if state.clusterAutoscaler.enabled {